src/bulma/pages/dataImport/Index.vue

```diff
@@ -87,6 +87,15 @@
                     {{ column.enum._get(row.status) }}
                 </span>
             </template>
+            <template v-slot:cancel="{ icon, row }">
+                <a class="button is-small is-table-button has-margin-left-small is-row-button has-text-danger"
+                    @click="cancelStuckImports(row)"
```

Comment:
`cancelStuckImports` => `cancel` (we will use it for other imports too, not only stuck)